### ISSUE TITLE
Download link

### ISSUE THREAD

**Issue by AzazelHD** [cheat-engine/cheat-engine#2660]
https://github.com/cheat-engine/cheat-engine/releases/latest

Where is the download link? Not the source code.

**Comment by fade2gray** [comment]
![Capture-3](https://github.com/cheat-engine/cheat-engine/assets/1332009/1d25b186-8f39-47b2-af4e-27c69f6b1b70)

**Comment by AzazelHD** [comment]
Yeah, and on that website i get an installer .exe that tries to install mcafee. Why trying to install bullshit programs? Is there more malware on it?

**Comment by fade2gray** [comment]
If you don't want the optional apps that are offered on the installer, you don't have to accept them.

There are three other choices you can make:

1. Download the [Patreon installer](https://www.patreon.com/cheatengine), which does not include the optional offers.
2. Download the source code and build Cheat Engine yourself using [these instructions](https://github.com/cheat-engine/cheat-engine#basic-build-instructions).
3. Do without.
